rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -843,15 +843,12 @@ MasterGraph::copy_out_tensor(void *out_ptr, RocalTensorFormat format, float mult
                                 fB = _mm256_fmadd_ps(fB, pmul0, padd0);
                                 fG = _mm256_fmadd_ps(fG, pmul1, padd1);
                                 fR = _mm256_fmadd_ps(fR, pmul2, padd2);
-                                _mm256_storeu_ps(tempB, fB);
-                                _mm256_storeu_ps(tempG, fG);
-                                _mm256_storeu_ps(tempR, fR);
-                                for(int cnt = 0; cnt < 8; cnt++)
-                                {
-                                    B_buf_16[cnt] = (half) tempB[cnt];
-                                    G_buf_16[cnt] = (half) tempG[cnt];
-                                    R_buf_16[cnt] = (half) tempR[cnt];
-                                }
+                                tempB = _mm256_cvtps_ph(fB, _MM_FROUND_NO_EXC);
```

Comment:
probably you need to specify rounding mode here (round towards zero)

Reply:
Used _MM_FROUND_TO_ZERO as rounding mode for rounding towards zero